modules/test/playwright/tests/roles-item-selector-web/rolesItemSelector.spec.ts

```diff
@@ -0,0 +1,74 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {loginTest} from '../../fixtures/loginTest';
+import {productMenuPageTest} from '../../fixtures/productMenuPageTest';
+import {rolesItemSelectorPagesTest} from '../../fixtures/rolesItemSelectorPagesTest';
+import {segmentEditorPagesTest} from '../../fixtures/segmentEditorPagesTest';
+import getRandomString from '../../utils/getRandomString';
+
+export const test = mergeTests(
+	loginTest(),
+	productMenuPageTest,
+	rolesItemSelectorPagesTest,
+	segmentEditorPagesTest
+);
+
+test('LPD-48734 Multiple roles can be added using the item selector', async ({
+	page,
+	productMenuPage,
+	rolesItemSelectorPage,
+	segmentEditorPage,
+}) => {
+	await productMenuPage.openProductMenuIfClosed();
+
+	await productMenuPage.goToSegments();
+
+	await page.getByRole('link', {name: 'Add New User Segment'}).click();
+
+	await page.getByText('No Conditions yet').waitFor();
```

Comment:
these are locators, they should go in some page

Reply:
@smottal , that is fair. FYI, this portion of the code was just copy/pasted from the segmentation team's playwright test.